### PR TITLE
Highlight mouse cursor even when no buttons are pressed

### LINKED ISSUE #1086: [Feature Request] Highlight mouse cursor even when no buttons are pressed
**Is your feature request related to a problem? Please describe.**
When presenting a software demo on a big screen (people looking at it from a distance), the mouse cursor in the demo can be hard to follow. The same can happen when looking at recordings on a phone.

**Describe the solution you'd like**
Add an option to highlight the mouse cursor without any buttons being pressed.
The existing mouse click feature already has all the mouse positions available, and can be extended with a new option for the highlight color. :)

**Describe alternatives you've considered**
Longer demos with more explanations. :(

**Additional context**
See pull request:
https://github.com/NickeManarin/ScreenToGif/pull/1046
image

### PR DESCRIPTION
**Pull request by zsee**
Hi!

This is a small change to the "Mouse Clicks" overlay feature: Adds a new setting to highlight the mouse cursor without clicks.
The name "Mouse Clicks" didn't seem appropriate anymore, so it's also renamed to "Mouse Events".

The highlight color is fully transparent by default, so it's not visible, and added a check to skip drawing transparent mouse overlays.

This is my first pull request, so I hope it's ok. :) Wasn't sure of the migrations part and what to do about the localization files.
Here are some screenshots:
image
image

closes #1086

### CODE REVIEW

**Comment by NickeManarin** [comment]
Hello @zsee, thank you for the PR.
It's important to first open an issue ticket to communicate your intention and to check if it fits with the idea of the project.
As well as so I can plan when to assign the new feature to a version.

Can you rebase your branch to the latest master?
And move the migration code to 2_36_0To2_37_0.

**Comment by vini123** [comment]
虚幻之物对应着冥冥之路！：《羽》

**Comment by zsee** [comment]
Hi @NickeManarin! 
Thanks for the feedback. You're absolutely right, I should have opened an issue first. :)
Opened it now, rebased, and linked the PR with the issue #1086

**Comment by NickeManarin** [comment]
Thanks!
